app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/ContributorsFragment.java

```diff
@@ -52,6 +52,13 @@ public void onAttach(Context context) {
     }
 
     @Override
+    public void onDetach() {
+        viewModel.setCallback(null);
```

Comment:
🙋
I guess the callback is set null in `viewModel#destroy()` like this.

```java
public void destroy() {
    this.callback = null;
}
```

Is it better to call `setCallback(null)` here? 🤔